source_files/main.c

```diff
@@ -37,20 +37,14 @@
 #include "watchdog.h"
 #include "help.h"
 
-$MAC /* Mac OS compatibility */
-$MAC #ifdef __MACH__
-$MAC #include <mach/mach_time.h>
-$MAC #define CLOCK_REALTIME 0
-$MAC #endif
-$MAC
 /*
  * Header for local functions
  */
 #include "work.h"
 #include "cpu.h"
-$CUDA #ifdef CUDA
-$CUDA #include "gpu.h"
-$CUDA #endif
+#ifdef CUDA
```

Comment:
This file looks like the generated main.c, not like the template source_files/main.c